.changeset/plenty-moles-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/hydrogen-react': patch
+---
+
+Shopify Analytics methods and components
```

Comment:
Can you list out the methods / functions that are now available? We don't necessarily need full on documentation, but it would be nice to at least know what new functions are available in the package.